### PR TITLE
Fix reliability in pig related genes ideogram (SCP-5293)

### PR DESCRIPTION
**Pull request by eweitz**
This fixes slowness and reliability issues in related genes for pig (_Sus scrofa_), which affects some recently-initialized studies atop production, by pulling in a small upstream fix.

Here's how the fix looks:

https://github.com/broadinstitute/single_cell_portal_core/assets/1334561/df912490-cfab-4d17-8195-c6d31bf7a273

<span id="test"></span>
### Test
To manually test:
* If local: 
  * `yarn install --force`, then `bin/vite dev` (i.e., don't run with service worker caching)
  * Update a local study with all human genes to use "pig" as the species, by going to "Upload/Edit Data" and changing the "Species" drop-down from "human" to "pig"
* If on staging: go to https://singlecell-staging.broadinstitute.org/single_cell/study/SCP280/pig-all-genes
* Search a gene, e.g. RAD51
* Ensure it displays as shown in video

This satisfies SCP-5293.

### CODE REVIEW

**Review by bistline — APPROVED** [review]
This works, but I had to do some yak shaving to get it configured correctly.  I hadn't uploaded a genome annotation for pig data, so I needed to add that (which required getting the `gtf` files from production and putting them in the correct reference data bucket for my local instance).  Also, it didn't work correctly until I updated both the processed _and_ raw matrix files to be `pig`.  Not sure why that mattered, but it did.

Once all of that was done, it worked correctly.  But I think this highlights the pain points of our species integration.

**Review by ehanna4 — APPROVED** [review]
Approving with plan to functional review in staging after merge

**Comment by ehanna4** [comment]
> Approving with plan to functional review in staging after merge

staging tested ✅

**Pull request merged**
